src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -126,26 +126,21 @@ public void parseAddress_validValueWithWhitespace_returnsTrimmedAddress() throws
     }
 
     @Test
-    public void parseEmail_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> ParserUtil.parseEmail((String) null));
-    }
-
-    @Test
-    public void parseEmail_invalidValue_throwsParseException() {
```

Comment:
To fix the coverage bug, try throwing a parse exception when a string of " " only is detected for remark